src/store/index.ts

```diff
@@ -27,6 +27,8 @@ interface TaxesState {
   colors: Colors;
   rnh: boolean;
   rnhTax: number;
+  solidarityFee1: number;
```

Comment:
Could you be a bit more explicit and name it `solidarityFeeAbove80k` and `solidarityFeeAbove250k` - or something like that?